new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -199,49 +258,451 @@ public void testDeleteOperatorButNotDomain() {
     }
 
     logger.info("Checking that the admin server pod state was not changed after the operator was deleted");
-    assertThat(podStateNotChanged(adminServerPodName, domainUid, domainNamespace, adminPodOriginalTimestamp))
-        .as("Test state of pod {0} was not changed in namespace {1}", adminServerPodName, domainNamespace)
+    assertThat(podStateNotChanged(adminServerPodName, domain1Uid,
+        domain1Namespace, adminPodOriginalTimestamp))
+        .as("Test state of pod {0} was not changed in namespace {1}", adminServerPodName, domain1Namespace)
         .withFailMessage("State of pod {0} was changed in namespace {1}",
-            adminServerPodName, domainNamespace)
+            adminServerPodName, domain1Namespace)
         .isTrue();
 
     // check that the states of managed server pods in the domain were not changed
     logger.info("Checking that the managed server pod state was not changed after the operator was deleted");
     for (int i = 1; i <= replicaCount; i++) {
       String managedServerPodName = managedServerPrefix + i;
-      assertThat(podStateNotChanged(managedServerPodName, domainUid, domainNamespace,
+      assertThat(podStateNotChanged(managedServerPodName, domain1Uid, domain1Namespace,
               managedServerPodOriginalTimestampList.get(i - 1)))
-          .as("Test state of pod {0} was not changed in namespace {1}", managedServerPodName, domainNamespace)
+          .as("Test state of pod {0} was not changed in namespace {1}",
+              managedServerPodName, domain1Namespace)
           .withFailMessage("State of pod {0} was changed in namespace {1}",
-              managedServerPodName, domainNamespace)
+              managedServerPodName, domain1Namespace)
           .isTrue();
     }
 
     // verify the sample app in the domain is still accessible from all managed servers through NGINX
     logger.info("Checking that the sample app can be accessed from all managed servers through NGINX "
         + "after the operator was deleted");
     verifySampleAppAccessThroughNginx();
+
   }
 
   /**
-   * TODO: remove this after Sankar's PR is merged
-   * The cleanup framework does not uninstall NGINX release. Do it here for now.
+   * Create operator and verify it is deployed successfully.
+   * Create a custom domain resource and verify all server pods in the domain were created and ready.
+   * Verify NGINX can access the sample app from all managed servers in the domain.
+   * Delete operator.
```

Comment:
Explain "Delete operator"  Are you deleting the Operator Pod or Changing the Replica to Zero.

Reply:
changed to uninstall

Reply:
Uninstall operator --> Uninstall operator helm chart.

Reply:
done